market_maker_keeper/dydx_market_maker_keeper.py

```diff
@@ -96,6 +96,8 @@ def __init__(self, args: list):
 
         self.market_info = self.dydx_api.get_markets()
 
+        self.all_pairs = ['WETH-DAI', 'WETH-USDC', 'DAI-USDC']
```

Comment:
Presumably, this isn't all the pairs traded on dYdX.  Is it all pairs which might be traded by this open source software?  Should we make it configurable, to maximize utility for market makers using this software?

Reply:
This is currently all of the traded pairs, but will work on making it passed in as an argument

Reply:
Instead of making it configurable, perhap's it'd be better to take it from a [markets](https://docs.dydx.exchange/#/api?id=get-v2markets) request?

Reply:
Good point, that makes more sense. Will adjust accordingly.